README.md

```diff
@@ -1,6 +1,65 @@
-## adaXT - Fast adaptable and extendable trees for research
+## Fast Adaptable and Extendable Trees for Research
 
-**adaXT** is a Python module for tree-based regression and classification that is fast, adaptable and extendable and aims to provide researchers a more flexible workflow when building tree-based models.
+**adaXT** is a Python module for tree-based machine-learning
+algorithms that is fast, adaptable and extendable and aims to provide
+researchers a more flexible workflow when developing tree-based
+models.
 
-### Installation
-adaXT is available at [pypi](https://pypi.org/) and can be installed by running ```pip install adaXT```.
+It is distributed under the [3-Clause BSD license](LICENSE).
```

Comment:
```suggestion
It is distributed under the [3-Clause BSD license](https://github.com/NiklasPfister/adaXT/blob/main/LICENSE).
```
As the license doesn't have it's own page, you can't use the default referencing and you have
to leave a link to the site itself.

Reply:
Why is the full url not needed in the current version?

Reply:
Not sure I follow. The current link doesn't work? If you do mkdocs serve, it will send you 
to a 404 page not found error.